### DISCUSSION TITLE
Is there any language package for rabbitmq_management?

### DISCUSSION THREAD

**Discussion by ilivecoding** [rabbitmq/rabbitmq-server#4464] [Other] [answered]
![图片](https://user-images.githubusercontent.com/7898334/162380529-029c568d-446c-4342-9f41-de978a2b4c4b.png)

loss SaveChanges

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ management UI does not offer any i18n customization, unfortunately.
